### DISCUSSION TITLE
NVFlare on Mobile

### DISCUSSION THREAD

**Discussion by LukeBauer** [NVIDIA/NVFlare#2375] [General]
Has NVFlare ever been used on mobile devices such as Android or IPhone, mainly as the client? If so, are there any resources to help me get started on doing so? If not, are there any plans to extend NVFlare to mobile architectures? Thank You!

**Comment by chesterxgchen** [comment]
@LukeBauer  we have no plan to support mobile devices. We focused on cross-silo use cases and some edge computing, but not FL on mobile devices

**Comment by brent8642** [comment]
NVFlare Day 2024 featured Flower.AI's partnership with NVIDIA. We are evaluating Flower for device FL while looking at NVFlare and NVFlare + Flower for cross silo.

**Reply by chesterxgchen** [reply]
let us know how it go
